source/framework/tools/src/TRestTools.cxx

```diff
@@ -606,6 +626,22 @@ string TRestTools::GetFileNameExtension(string fullname) {
     return fullname;
 }
 
+///////////////////////////////////////////////
+/// \brief Gets the filename root as the substring found before the lastest "."
+///
+/// Input: "/home/jgalan/abc.txt" Output: "abc"
+///
+string TRestTools::GetFileNameRoot(string fullname) {
+    int pos1 = fullname.find_last_of('/', -1);
+    int pos2 = fullname.find_last_of('.', -1);
```

Comment:
Please use appropriate data types, in this case `size_t`, I have created a suggestion via PR about this https://github.com/rest-for-physics/framework/pull/197

Reply:
I see, but why you dont just make a change proposal at this PR?

Reply:
OK, I though, this was the way to create it, now I have added the suggestion, ignore this PR